app/authentication/session_storage.py

```diff
@@ -1,144 +1,111 @@
-from uuid import uuid4
-
-from flask import session
 from structlog import get_logger
-
 from app.data_model.models import EQSession, db
+from app.data_model.session_data import SessionData
+from app.settings import EQ_SESSION_ID
+from flask import session as cookie_session
+import simplejson as json
 
-USER_ID = 'user_id'
-USER_IK = 'user_ik'
-SURVEY_COMPLETED_METADATA = 'survey_completed_metadata'
-EQ_SESSION_ID = 'eq-session-id'
 
 logger = get_logger()
 
 
 class SessionStorage:
 
-    @staticmethod
-    def store_user_id(user_id):
+    def __init__(self):
+        self.eq_session = None
```

Comment:
I think we need to simplify this:
- The class should expose the properties` user_id`, `eq_session_id` and `session_data` - this will remove the need for the `get_user_id` and `get_session_data` properties. The `session_data` property should be a `SessionData` class.
- We should just need `load`, `save` and `delete` methods. To update a session we would update the `session_data` property and call save.
- The eq_session_id from the cookie should be passed in, rather than having a dependency on the flask cookie session in this class

Also, we should rename this file session_store.py and move it alongside questionnaire_store.

Reply:
i don't think we need `load`. We should do that on the init as we do with `QuestionnaireStore`